main.py

```diff
@@ -210,10 +210,10 @@ def update(self) -> Image:
     Image(
         registry='registry-1.docker.io',
         source='library/postgres',
-        tag='14-alpine',
+        tag='14-alpine',  # 14.11
```

Comment:
I wonder if it makes sense to instead include the minor version here (and then use that in sentry) -- a slightly less floating tag

Reply:
I can add it as an additional image so we don't unexpectedly break anything else that refers to 14-alpine.